build/pause/Dockerfile

```diff
@@ -15,6 +15,6 @@
 ARG BASE
 FROM ${BASE}
 ARG ARCH
-ADD bin/pause-linux-${ARCH} /pause
 USER 65535:65535
-ENTRYPOINT ["/pause"]
+COPY --chown=65535:65535 --chmod=555 bin/pause-linux-${ARCH} /pause
```

Comment:
Wouldn't `--chmod 555` allow *any* user to read and execute the binary? If so,  `--chown` looks redundant to me.